core/packages/PackageInfo.h

```diff
@@ -41,7 +41,7 @@ struct VisibleTo {
     VisibleToType visibleToType;
 
     VisibleTo(std::vector<core::NameRef> packageName, VisibleToType visibleToType)
-        : packageName(packageName), visibleToType(visibleToType){};
+        : packageName(std::move(packageName)), visibleToType(visibleToType){};
```

Comment:
I'm not sure if we can `move` here. The constructor is called [here](https://github.com/sorbet/sorbet/blob/master/packager/packager.cc#L397). If we add this `move`, then the `visibleTo_` instance variable will no longer be valid.

Reply:
The constructor takes the vector by value, so that call site does a copy into a fresh instance.